client/app/billing/autoRenew/autorenew.html

```diff
@@ -98,6 +98,10 @@
             <extra-top>
                 <oui-action-menu data-text="{{:: 'autorenew_service_action_title' | translate }}">
                     <div data-ng-if="$ctrl.selectedServices.length > 0">
+                        <oui-action-menu-item data-href="{{ $ctrl.manualRenewUrl }}"
```

Comment:
```suggestion
                        <oui-action-menu-item data-href="{{:: $ctrl.manualRenewUrl }}"
```

Reply:
No, the `manualRenewUrl` can change, a one time binding won't update the href if you select more services :)